packages/discovery-react-components/src/components/SearchResults/SearchResults.tsx

```diff
@@ -125,6 +139,8 @@ const SearchResults: React.FunctionComponent<SearchResultsProps> = ({
     typeof showTablesOnlyToggle === 'undefined' ? hasTables : showTablesOnlyToggle
   );
 
+  useUpdateQueryReturnParam({ displaySettings, resultLinkField });
```

Comment:
This is another BREAKING CHANGE, since we are changing what the parent context requests when doing a search.